### DISCUSSION TITLE
Option for exclude certain file extensions from Bob build

### DISCUSSION THREAD

**Discussion by rusiruavb** [callstack/react-native-builder-bob#334] [Q&A]
I'm developing a react native application that have `.native.tsx`, `.android.tsx`, `.ios.tsx` for mobile and `.web.tsx` for web application. When I build the application using `bob build` command, it is compile both web and mobile code into one build. But what I need to do is exclude the `.web.tsx` files from the mobile build. *Is there any specify way to handle this kind of situation?* Following is my **react-native-builder-bob** configurations in the package.json.

```
  "react-native-builder-bob": {
    "source": "src",
    "output": "lib",
    "targets": [
      "commonjs",
      "module",
      [
        "typescript",
        {
          "project": "tsconfig.build.json"
        }
      ]
    ]
  }
```

**Comment by satya164** [comment]
> it is compile both web and mobile code into one build. But what I need to do is exclude the `.web.tsx` files from the mobile build

What do you mean by that? There's no "web build" and "mobile build".
